one_fm/api/api.py

```diff
@@ -152,41 +158,62 @@ def push_notification(employee_id, title, body, checkin, arriveLate ,checkout ):
                 token=registration_token,
             )
         response = messaging.send(message)
-        print(response)
     return response
 
-# This function is used to send notification through Firebase CLoud Message. 
-# It is a rest API that sends request to "https://fcm.googleapis.com/fcm/send"
-# Params: employee_id e.g. HR_EMP_00001, , title:"Title of your message", body:"Body of your message"
-# bench execute --kwargs "{'employee_id':'HR-EMP-00001','title':'Hello','body':'Testing'}" one_fm.api.api.push_notification_rest_api
 @frappe.whitelist()
-def push_notification_rest_api(employee_id, title, body):
-    """
+def push_notification_rest_api(employee_id, title, body, checkin, arriveLate ,checkout ):

```

Comment:
Same here. Follow appropriate naming for the method. It can be misleading to other developers trying to use the same method to send push notifications.